2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,16 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    let averageArr=[];
+    for (let closingPricesForStock of closingPricesForAllStocks){
+        let sum =0;
+        for (let item of closingPricesForStock){
+            sum=sum+item;
```

Comment:
this could be a great opportunity to use a reduce function. where your accumulator is the sum